contracts/CourtRoom.sol

```diff
@@ -37,7 +45,6 @@ contract CaseContract is Owned {
 		return clm.claimId;
 	}
 
-
 	function submitEvidence(bytes32 _claimId,bytes32 _evident) returns (uint status) {
 
 		// Maximum amount of evidence has been submitted
```

Comment:
Implementation detail in fairness. But the idea was to limit the amount of evidence in the attempt of standardising the judgement process.